tests/snapshots/tests.contrib.starlette.test_starlette.test_subapp_snapshot.json

```diff
@@ -0,0 +1,43 @@
+[[
+  {
+    "name": "starlette.request",
+    "service": "starlette",
+    "resource": "GET /sub-app/hello/{name}",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "web",
+    "meta": {
+      "http.method": "GET",
+      "http.status_code": "200",
+      "http.url": "http://testserver/sub-app/hello/name",
+      "http.version": "1.1",
+      "runtime-id": "9e83e62bb7684737bb8c4aed8e8a0614"
+    },
+    "metrics": {
+      "_dd.agent_psr": 1.0,
+      "_dd.top_level": 1,
+      "_dd.tracer_kr": 1.0,
+      "_sampling_priority_v1": 1,
+      "system.pid": 71492
+    },
+    "duration": 271000,
+    "start": 1653313985950011000
+  },
+     {
+       "name": "starlette.request",
+       "service": "starlette",
+       "resource": "GET /hello/name",
```

Comment:
Is this supposed to be `GET /hello/{name}` ?

This might mean the ASGI middleware is being used instead of the newer code?

Reply:
```suggestion
       "resource": "GET /hello/{name}",
```